<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="crate_base_label">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="PyDMLabel" name="PyDMLabel_9">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Global Control</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PyDMPushButton" name="all_off_btn">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>All Off</string>
       </property>
       <property name="channel" stdset="0">
        <string>ca://TMO:PRO2:MPOD:01:Crate:GlobalOnOff</string>
       </property>
       <property name="pressValue" stdset="0">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PyDMPushButton" name="all_on_btn">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>All On</string>
       </property>
       <property name="channel" stdset="0">
        <string>ca://TMO:PRO2:MPOD:01:Crate:GlobalOnOff</string>
       </property>
       <property name="pressValue" stdset="0">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="PyDMCheckbox" name="PyDMCheckbox">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Set Global Kill</string>
       </property>
       <property name="channel" stdset="0">
        <string>ca://TMO:PRO2:MPOD:01:Crate:GlobalEmergency</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="PyDMLabel" name="PyDMLabel">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Module 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PyDMEmbeddedDisplay" name="PyDMEmbeddedDisplay">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>45</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="macros" stdset="0">
        <string>MODULE=TMO:PRO2:MPOD:01:M0</string>
       </property>
       <property name="filename" stdset="0">
        <string>mpod_module.ui</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="PyDMLabel" name="PyDMLabel_2">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Module 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PyDMEmbeddedDisplay" name="PyDMEmbeddedDisplay_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>45</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="macros" stdset="0">
        <string>MODULE=TMO:PRO2:MPOD:01:M1</string>
       </property>
       <property name="filename" stdset="0">
        <string>mpod_module.ui</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="PyDMLabel" name="PyDMLabel_3">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Module 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PyDMEmbeddedDisplay" name="PyDMEmbeddedDisplay_3">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>45</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="macros" stdset="0">
        <string>MODULE=TMO:PRO2:MPOD:01:M2</string>
       </property>
       <property name="filename" stdset="0">
        <string>mpod_module.ui</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="PyDMLabel" name="PyDMLabel_4">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Module 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PyDMEmbeddedDisplay" name="PyDMEmbeddedDisplay_4">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>45</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="macros" stdset="0">
        <string>MODULE=TMO:PRO2:MPOD:01:M3</string>
       </property>
       <property name="filename" stdset="0">
        <string>mpod_module.ui</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="PyDMLabel" name="PyDMLabel_5">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Module 4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PyDMEmbeddedDisplay" name="PyDMEmbeddedDisplay_5">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>45</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="macros" stdset="0">
        <string>MODULE=TMO:PRO2:MPOD:01:M4</string>
       </property>
       <property name="filename" stdset="0">
        <string>mpod_module.ui</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="PyDMLabel" name="PyDMLabel_6">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Module 5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PyDMEmbeddedDisplay" name="PyDMEmbeddedDisplay_6">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>45</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="macros" stdset="0">
        <string>MODULE=TMO:PRO2:MPOD:01:M5</string>
       </property>
       <property name="filename" stdset="0">
        <string>mpod_module.ui</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="PyDMLabel" name="PyDMLabel_7">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Module 6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PyDMEmbeddedDisplay" name="PyDMEmbeddedDisplay_7">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>45</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="macros" stdset="0">
        <string>MODULE=TMO:PRO2:MPOD:01:M6</string>
       </property>
       <property name="filename" stdset="0">
        <string>mpod_module.ui</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="PyDMLabel" name="PyDMLabel_8">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Module 7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PyDMEmbeddedDisplay" name="PyDMEmbeddedDisplay_8">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>45</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="macros" stdset="0">
        <string>MODULE=TMO:PRO2:MPOD:01:M7</string>
       </property>
       <property name="filename" stdset="0">
        <string>mpod_module.ui</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="PyDMShellCommand" name="PyDMShellCommand">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>VMI Panel</string>
       </property>
       <property name="showIcon" stdset="0">
        <bool>false</bool>
       </property>
       <property name="commands" stdset="0">
        <stringlist>
         <string>pydm /cds/group/pcds/epics-dev/aegger/screens/tmo/mpod/mpod_vmi.ui</string>
        </stringlist>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMLabel</class>
   <extends>QLabel</extends>
   <header>pydm.widgets.label</header>
  </customwidget>
  <customwidget>
   <class>PyDMCheckbox</class>
   <extends>QCheckBox</extends>
   <header>pydm.widgets.checkbox</header>
  </customwidget>
  <customwidget>
   <class>PyDMEmbeddedDisplay</class>
   <extends>QFrame</extends>
   <header>pydm.widgets.embedded_display</header>
  </customwidget>
  <customwidget>
   <class>PyDMPushButton</class>
   <extends>QPushButton</extends>
   <header>pydm.widgets.pushbutton</header>
  </customwidget>
  <customwidget>
   <class>PyDMShellCommand</class>
   <extends>QPushButton</extends>
   <header>pydm.widgets.shell_command</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
